src/invitations/invitations-list.js

```diff
@@ -0,0 +1,26 @@
+import { FirebaseList } from "src/firebase";
+import * as InvitationActions from "./actions";
+import { Record } from "immutable";
+
+export const InvitationList = new Record({
+  id: null,
+  created: null,
+  updated: null,
+  projectId: null,
+  name: null,
+  creatorId: null,
+  creator: null,
+  url: null,
+  canAdd: [],
+  canView: [],
+  canDelete: []
```

Comment:
Actually there is also `canComment` but we can assume that if a user can Add task he can add comment. 
Also as to `canView`
We have 2 types of projects - Private where this is important and public where this is probably won't be used - as public allows anyone to view it.
As to burnerot it would be a public project where all can view it

Reply:
The invitation list permissions are related only to the specific invitation list object, not to the tasks. Each task should be with the same permissions fields so we can manage it easily by adding/removing ids from every entity.